package.json

```diff
@@ -4,8 +4,7 @@
     "version": "1.0.0",
     "scripts": {
         "clean": "rm -rf packages/**/build/",
-        "lerna-bootstrap": "yarn install && lerna bootstrap --ci",
```

Comment:
Why no more lerna-bootstrap ?

Reply:
It's not needed for now and i don't know why it doesn't work on my machine :/